vms/administrator/tests/test_formFields.py

```diff
@@ -73,11 +73,35 @@ def check_event_form_values(self, event):
         :param event: Iterable consisting values for events.
         """
         settings = self.settings
+        try:
```

Comment:
Instead of having this method multiple times, can you make a separate method and call this method everywhere.